spec/solargraph/rspec/convention_spec.rb

```diff
@@ -598,6 +599,87 @@ class MyClass; end
         'Class'
       )
     end
+
+    context 'on variables' do
+      context 'in an example' do
```

Comment:
minor: These two feel a bit redundant, can we only have only one, context/describe block, for example:

```suggestion
     context/describe 'local variables' do
      context 'in an example' do
```

Reply:
Sounds good - I initially planned on writing more specs but as I mentioned above, it's hard to test for this. A lot of stuff already worked before the changes, but anything programmatic broke it, which is why I noticed it initially